calibration/ECalDigitisation_ContainedEvents.cc

```diff
@@ -519,14 +537,28 @@ bool ParseCommandLine(int argc, char *argv[], ECalDigitisation &eCalDigitisation
         case 'e':
             eCalDigitisation.m_fitPercentage = atof(optarg);
             break;
+        case 'g':
+            eCalDigitisation.m_element = optarg;
+            break;
+        case 'i':
+            eCalDigitisation.m_lowerCosThetaCut = atof(optarg);
+            break;
+        case 'j':
+            eCalDigitisation.m_upperCosThetaCut = atof(optarg);
+            break;
         case 'f':
+        case 'h':
+        case 'k':
         default:
             std::cout << std::endl << "Calibrate " << std::endl
                       << "    -a        (mandatory, input file name(s), can include wildcards if string is in quotes)           " << std::endl
                       << "    -b value  (mandatory, true energy of photons being used for calibration)                          " << std::endl
                       << "    -c value  (optional, fractional accuracy to calibrate CalibrECAL to, default 0.02)                " << std::endl
                       << "    -d        (mandatory, output path to send results to)                                             " << std::endl
                       << "    -e value  (optional, fit percentage used for calibration, default 90% of data with narrowest rms) " << std::endl
+                      << "    -g        (mandatory, element of the detector being calibrated (Barrel or EndCap))                " << std::endl
+                      << "    -i value  (mandatory, lower abs cos theta cut defining element, default 0)                        " << std::endl
```

Comment:
If this parameter is mandatory then there is no default value, maybe rather "minimum value 0"? Maximum value 1 in the next line.